src/Rx/Weno/TransmitProperties.php

```diff
@@ -60,41 +59,53 @@ public function createJsonObject()
         if ($this->subscriber == 'self') {
             $relationship = 'Y';
         } else {
-            $relationship = 'N';
+            $relationship = 'Y';
         }
```

Comment:
the above if conditional is always going to be `$relationship = 'Y'` no matter what

Reply:
they had me changing so many things over and over. It was difficult trying to keep up. So, I am just hard coding this to yes instead of pulling it from the system.